qa/tasks/scripts/ses_rack_dc_region_unavailability.sh

```diff
@@ -137,15 +137,15 @@ done
 # bring down rack
 for node2fail in ${rack4_hosts[@]}
 do
-    iptables_drop ${node2fail}.teuthology
+    iptables_drop ${node2fail}.*
```

Comment:
Why do you think it should work? The wildcard should be used gently, because in bash they are can be replaced with something in localfile system, will this work if you change it like:
```
    iptables_drop "${node2fail}.*"
```

Reply:
Yes. Done